projects/portfolio/portfolio-webpage-1.md

```diff
@@ -73,13 +73,13 @@ You should write clear + informative commit messages for each commit and create
 
 **Create your project**
 
-- [ ] Use bash to create a local portfolio-website directory. Use the command line to make an portfolio.html and an portfolio.css in the portfolio-website directory.
-- [ ] Use the command line to change the names of portfolio.html and portfolio.css to index.html and index.css.
-- [ ] Use the command line to make a gitignore file in your desktop directory and add at least one real unwanted file to the gitignore.
-- [ ] Use the command line to move the gitignore file from your desktop directory to your portfolio-website directory.
-- [ ] Using your command line, initialize a GitHub repo in portfolio-website.
-- [ ] Create a remote repo for the project in your GitHub account.
-- [ ] Using your command line, add and commit your portfolio-website directory, and then do a git push.
+- [ ] Use bash to create a local portfolio-website directory. Use the command line to make a portfolio.html file and a portfolio.css file in the portfolio-website directory.
+- [ ] Use the command line to change the names of portfolio.html and portfolio.css to index.html and styles.css.
+- [ ] Use the command line to make a .gitignore file in your desktop directory and add at least one real unwanted file to the .gitignore list.
+- [ ] Use the command line to move the .gitignore file from your desktop directory to your portfolio-website directory.
+- [ ] Using your command line, initialize a GitHub repo in portfolio-website directory.
+- [ ] Create a portfolio-website remote repository on Github. (DO NOT add a readME or .gitignore file in the set up)
+- [ ] Using your command line, git add and git commit your portfolio-website directory, and then do a git push.
```

Comment:
```suggestion
- [ ] Using your command line, add and commit your portfolio-website directory, and then do a git push.
```

The idea is to give context, but not a detailed walk through. They should be making the connections of using git for this instruction

Reply:
@daaimah123 Duly noted! I shall revert the original sentence.

Reply:
done!